api/docs/docs.go

```diff
@@ -151,6 +151,145 @@ const docTemplate = `{
+                        "type": "integer"
+                    }
+                ],
+            },
+        },
+        "/activities/{id}": {
+            "get": {
+                tags: ["activity"],
+                "description": "IDで指定されたactivitiesの取得",
+                "parameters": [
+                    {
+                        "name": "id",
+                        "in": "path",
+                        "description": "id",
+                        "required": true,
+                        "type": "integer"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "activitiesの取得",
+                    }
+                }
+            },
+            "put": {
+                tags: ["activity"],
+                "description": "activitiesの更新",
+                responses: {
+                    "200": {
+                        "description": "更新されたactivitiesが返ってくる",
+                    }
+                },
+                "parameters": [
+                    {
+                        "name": "id",
+                        "in": "path",
+                        "description": "id",
+                        "required": true,
+                        "type": "integer"
+                    },
+                    {
+                        "name": "sponsor_style_id",
+                        "in": "query",
+                        "description": "sponsor_style_id",
+                        "required": true,
+                        "type": "integer"
+                    },
+                    {
+                        "name": "user_id",
+                        "in": "query",
+                        "description": "user_id",
+                        "type": "integer"
+                    },
+                    {
+                        "name": "is_done",
+                        "in": "query",
+                        "description": "is_done",
+                        "type": "boolean"
+                    },
+                    {
+                        "name": "sponsor_id",
+                        "in": "query",
+                        "description": "sponsor_id",
+                        "type": "integer"
+                    },
+                ],
+            },
+            "delete": {
+                tags: ["activity"],
+                "description": "IDを指定してactivitiesの削除",
+                "parameters": [
+                    {
+                        "name": "id",
+                        "in": "path",
+                        "description": "id",
+                        "required": true,
+                        "type": "integer"
+                    }
+                ],
+                responses: {
+                    "200": {
+                        "description": "activitiesの削除完了",
+                    }
+                },
+            },
+        },
+        "/get_activities_for_view": {
```

Comment:
```suggestion
        "/activities/details": {
```
になるように、routerも修正してもらえるとgood